packages/vanchor-client/src/queries/deposit/depositDayInterval.ts

```diff
@@ -0,0 +1,261 @@
+    query,
+    {},
+    {
+      subgraphUrl,
+    }
+  );
+
+  return result.data.vanchorDepositEveryDays?.map((item: any) => {
+    return {
+      deposit: +item?.deposit,
+      subgraphUrl: subgraphUrl,
+      date: DateUtil.fromEpochToDate(parseInt(item?.date)),
+      vAnchorAddress: item?.vAnchorAddress,
+    };
+  });
+};
+
+export const GetVAnchorDepositByChainsDayInterval = async (
+  subgraphUrls: Array<SubgraphUrl>,
+  vAnchorAddress: string,
+  date: Date
+): Promise<Array<DepositByChainDayIntervalItem>> => {
+  const promises: Array<Promise<DepositByChainDayIntervalItem>> = [];
+
+  for (const subgraphUrl of subgraphUrls) {
+    promises.push(
+      GetVAnchorDepositByChainDayInterval(subgraphUrl, vAnchorAddress, date)
+    );
+  }
+
+  return await Promise.all(promises);
+};
+
+export const GetVAnchorsDepositByChainDayInterval = async (
+  subgraphUrl: SubgraphUrl,
+  vanchorAddresses: Array<string>,
+  date: Date
+): Promise<Array<DepositByVAnchorDayIntervalItem>> => {
+  const query = /* GraphQL */ `
+    query TotalValueLockedByVAnchor {
+      vanchorDepositEveryDays(
+        where: {
+          date: "${DateUtil.fromDateToEpoch(date)}",
+          vAnchorAddress_in: [
+            ${vanchorAddresses
+              .map((address) => '"' + address.toLowerCase() + '"')
+              .join(',')}
+          ]
+        }
+      ) {
+        id
+        deposit
+        vAnchorAddress
+        date
+      }
+    }
+  `;
+  const result = await execute(
+    query,
+    {},
+    {
+      subgraphUrl,
+    }
+  );
+
+  const depositMap: { [vanchorAddress: string]: number } = {};
+
+  result.data.vanchorDepositEveryDays?.map((item: any) => {
+    if (!depositMap[item?.vAnchorAddress]) {
+      depositMap[item?.vAnchorAddress] = 0;
+    }
+
+    depositMap[item?.vAnchorAddress] += +item?.deposit;
+  });
+
+  const depositByVAnchorDayIntervalItems: Array<DepositByVAnchorDayIntervalItem> =
+    [];
+
+  for (const key in depositMap) {
+    depositByVAnchorDayIntervalItems.push({
+      vAnchorAddress: key,
+      deposit: depositMap[key],
+    });
+  }
+
+  return depositByVAnchorDayIntervalItems;
+};
+
+export const GetVAnchorsDepositByChainsDayInterval = async (
+  subgraphUrls: Array<SubgraphUrl>,
+  vanchorAddresses: Array<string>,
+  date: Date
+): Promise<Array<Array<DepositByVAnchorDayIntervalItem>>> => {
+  const promises: Array<Promise<Array<DepositByVAnchorDayIntervalItem>>> = [];
+
+  for (const subgraphUrl of subgraphUrls) {
+    promises.push(
+      GetVAnchorsDepositByChainDayInterval(subgraphUrl, vanchorAddresses, date)
+    );
+  }
+
+  return await Promise.all(promises);
+};
+
+export const GetVAnchorDepositByChainAndByTokenDayInterval = async (
+  subgraphUrl: SubgraphUrl,
+  vAnchorAddress: string,
+  tokenSymbol: string,
+  date: Date
+): Promise<Array<DepositByChainAndByTokenDayIntervalItem>> => {
+  const query = /* GraphQL */ `
+    query MyQuery {
+      vanchorDepositByTokenEveryDays(
```

Comment:
Same here, I think this naming sounds awkward. Is this what the 15 min looks like?